intel_extension_for_transformers/transformers/modeling/modeling_auto.py

```diff
@@ -204,7 +204,8 @@ def from_pretrained(cls, pretrained_model_name_or_path, *model_args, **kwargs):
         elif kwargs.get("use_neural_speed", None) is not None:
             use_neural_speed = kwargs.pop("use_neural_speed", True) and not use_xpu
         else:
-            config = transformers.AutoConfig.from_pretrained(pretrained_model_name_or_path)
+            config = transformers.AutoConfig.from_pretrained(pretrained_model_name_or_path,
+                                                             trust_remote_code = kwargs.get('trust_remote_code', False))
             if hasattr(config, "model_type") == False:

```

Comment:
could you use transformers.AutoConfig.from_pretrained(pretrained_model_name_or_path, **kwargs)